lib/vagrant-sshfs/cap/linux/sshfs_mount.rb

```diff
@@ -142,6 +143,12 @@ def self.sshfs_slave_mount(machine, opts, hostpath, expanded_guest_path)
           r1, w1 = IO.pipe # reader/writer from pipe1
           r2, w2 = IO.pipe # reader/writer from pipe2
 
+          # For issue #27 we'll need to create a tmp files for STDERR
```

Comment:
#27, as reported stems from the fact that the STDERR stream was never closed and was still sending output to the terminal. Using a tmpfile instead means it is sending it somewhere else instead, and the installer doesn't get caught up on that fact.